e2e/jest.config.js

```diff
@@ -28,21 +31,11 @@ module.exports = {
     "/node_modules/",
     "/tsc-out/"
   ],
-  "testMatch": [
-    "<rootDir>/tests/**/*.spec.ts"
-  ],
+  testMatch: TEST_MODE === "nightly-integration" ? [ "<rootDir>/tests/nightly/**/*.spec.ts", ] : [ "<rootDir>/tests(?!\/nightly)/**/*.spec.ts", ],
   "transformIgnorePatterns": [
     "<rootDir>/node_modules/(?!tests)"
   ],
-  "moduleFileExtensions": [
-    "ts",
-    "spec.ts",
-    "tsx",
-    "js",
-    "jsx",
-    "json",
-    "node"
-  ],
+  "moduleFileExtensions": [ ...defaults.moduleFileExtensions ],
```

Comment:
`moduleFileExtensions ` default array is: ["js", "json", "jsx", "ts", "tsx", "node"]